infra-parent/infra-v2/src/main/java/com/redhat/service/smartevents/infra/v2/api/models/processors/ProcessorDefinition.java

```diff
@@ -0,0 +1,47 @@
+package com.redhat.service.smartevents.infra.v2.api.models.processors;
+
+import java.util.Objects;
+
+import javax.validation.constraints.NotNull;
+
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+public class ProcessorDefinition {
+
+    @JsonProperty("flows")
+    @NotNull(message = "Processor flows cannot be null")
+    // ObjectNode is not rendered properly by swagger
+    @Schema(implementation = Object.class, required = true)
+    private ObjectNode flows;
```

Comment:
@r00ta "wrapper" for Camel's `flows` (useful for if we need to persist additional meta-data later).